test/conductors/test_cloudtasks_conductor.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import googleapiclient
+import io
+import mock
+import os
+import subprocess
+import unittest
+import uuid
+
+from artman.cli import main
+from artman.conductors import cloudtasks_conductor
+
+
+class ConductorTests(unittest.TestCase):
+
+    _FAKE_PULL_TASKS_RESPONSE = {
+        'tasks': [
+            {
+                'name': 'projects/foo/locations/bar/queues/baz/abcde',
+                'pullTaskTarget': {
+                    # Decoded string is "--api pubsub --lang python"
+                    'payload': 'LS1hcGkgcHVic3ViIC0tbGFuZyBweXRob24='
+                    },
+                'scheduleTime': '',
+                'view': 'FULL'
+            }
+        ]
+    }
+
+    _FAKE_QUEUE_NAME = 'projects/foo/locations/bar/queues/baz'
+
+    @mock.patch.object(cloudtasks_conductor, '_create_tasks_client')
+    @mock.patch.object(cloudtasks_conductor, '_prepare_dir')
+    @mock.patch.object(cloudtasks_conductor, '_pull_task')
+    @mock.patch.object(main, 'main')
+    @mock.patch.object(cloudtasks_conductor, '_ack_task')
+    @mock.patch.object(cloudtasks_conductor, '_cleanup_tmp_dir')
+    def test_start_conductor_succeed(self, cleanup_tmp_dir, ack_task, cli_main,
```

Comment:
I get really nervous when I see six mocks on a unit test. It is not _wrong_, but is there a better approach? (If the answer is no, that is fine.)